paddle/fluid/platform/device_context.cc

```diff
@@ -407,17 +407,32 @@ thread_local size_t cur_mkldnn_session_id = kMKLDNNSessionID_Default;
 // - For fixed-shape, it's a null string in default.
 // - For dynamic-shape, it's user specific.
 thread_local std::string cur_input_shape_str = "";
+// the cache size of different input shapes for MKLDNN.
+// Default 1 means fixed input shape, not dynamic shape.
+thread_local int cur_input_shape_cache_size = 1;
```

Comment:
how about cur_input_shape_cache_capacity instead of size?  size here is a kind of real data size, while it is for max cache size in my understanding.

Reply:
Got it. I will change.